src/logger/index.ts

```diff
@@ -2,6 +2,9 @@ import bunyan from 'bunyan';
 import bunyanDebugStream from 'bunyan-debug-stream';
 import { SentryStream } from 'bunyan-sentry-stream';
 import { Client as RavenClient } from 'raven';
+// type error when using import
+// https://github.com/googleapis/nodejs-logging-bunyan/issues/241
+const { LoggingBunyan } = require('@google-cloud/logging-bunyan');
```

Comment:
Could you move this line below other imports? I feel like a `require` statement doesn't belong here.